src/bin/psql/copy.c

```diff
@@ -157,6 +157,14 @@ parse_slash_copy(const char *args)
 
 		if (!token)
 			goto error;
+
+        xstrcat(&result->before_tofrom, " ");
+        xstrcat(&result->before_tofrom, token);
+        token = strtokx(NULL, whitespace, NULL, "\"",
```

Comment:
What happens if table name contains "."?

Reply:
It already has processed "." in table name.